lib/constants.js

```diff
@@ -27,6 +27,7 @@
 // Deprecation Code: DEP0008
 const constants = process.binding('constants');
 Object.assign(exports,
+              constants.os.dlopen,
```

Comment:
I don't think we should add things to something that's already deprecated.

Reply:
This has already been discussed, on this same PR-thread. I'll let you do the digging. (BTW: this is where mailing lists outpass github PRs: it's so much harder to dig discussion history).

Reply:
Okay, in that case SGTM. Sorry about not reading through the entire thread – though in my defense it is quite long.